api/features/models.py

```diff
@@ -970,3 +979,52 @@ def get_update_log_message(self, history_instance) -> typing.Optional[str]:
 
     def _get_environment(self) -> typing.Optional["Environment"]:
         return self.feature_state.environment
+
+
+class FeatureExport(models.Model):
+    """
+    Stores the representation of an environment's export of
+    features between the request for the export and the
+    ultimate download. Records are deleted automatically after
+    a waiting period.
+    """
+
+    # The environment the export came from.
+    environment = models.ForeignKey(
+        "environments.Environment",
+        related_name="feature_imports",
+        on_delete=models.CASCADE,
+        swappable=True,
+    )
+
+    # This is a JSON string of data used for file download
+    # once the task has completed assembly.
+    data = models.CharField(max_length=MAX_FEATURE_EXPORT_SIZE)
+    created_date = models.DateTimeField("DateCreated", auto_now_add=True)
```

Comment:
Ah, I expect you got this from another `created_date` in the application, but I really dislike renaming the column to `DateCreated`. Can we use just `created_at` here as this is the new standard.

Reply:
Yes that's what I did. I strongly prefer `created_at` too. I will update it.